src/main/java/seedu/address/logic/commands/listcommand/ListMentorCommand.java

```diff
@@ -21,6 +21,7 @@ public CommandResult execute(Model model) {
 
         this.displayMentors(model);
 
+        model.updateHistory();
```

Comment:
i dont think we need to update history after a list command right? We're not changing the state of anything

Reply:
It doesn't change the state of the data. But the thing is that if a user invokes the "undo" command immediately after a "list" command, he would be trying to undo the list command. Because unless we keep track of the commands that have been executed, I wouldn't know which state to revert back to. So right now, there is a 1-1 mapping between number of commands executed and the number of states that are stored.